index.js

```diff
@@ -26,19 +27,23 @@ const storage = new Storage({
 const metaStorage = new Storage()
 
 // Define the addon
-const addon = new addonSDK(manifest)
+function addon(options) {
+	options = options || {}
+	const addonBuilder = new addonSDK(options.disableCatalogSupport ? manifestNoCatalogs : manifest)
```

Comment:
creating an additional file for the manifest could be avoided like that:

```javascript
const { catalogs, ...manifestNoCatalogs } = manifest;
const addonBuilder = new addonSDK(options.disableCatalogSupport ? manifestNoCatalogs : manifest);
```

Reply:
It is not that simple. The `catalog` should also be removed from resources. Also the add-on name is changed to avoid confusion.